src/preview/withPseudoState.ts

```diff
@@ -82,10 +82,10 @@ const applyParameter = (rootElement: Element, parameter: PseudoStateConfig = {})
 // Shadow DOM can only access classes on its host. Traversing is needed to mimic the CSS cascade.
 const updateShadowHost = (shadowHost: Element) => {
   const classnames = new Set<string>()
-  // Keep any existing "pseudo-*" classes
+  // Keep any existing "pseudo-*" classes (but not "pseudo-*-all")
```

Comment:
Would be good to include a comment on _why_ we clear the `-all` classes (like you did in the PR description).